src/sentry/assistant/guides.py

```diff
@@ -3,6 +3,24 @@
 from django.utils.translation import ugettext_lazy as _
 
 
+# Guide Schema
+# id (text, required): unique id
+# cue (text, optional):  The text used to prmopt the user to initiate the guide.
+#                        Leaving this blank
```

Comment:
?

Reply:
i think we were discussing something, but i can't remember the utility of empty cues.